### ISSUE TITLE
error on fastly 

### ISSUE THREAD

**Issue by 6uu1** [serverless-dns/serverless-dns#130]
![image](https://user-images.githubusercontent.com/122669698/216825239-ba6b6363-fd05-4676-8de9-6c6c09889fb5.png)

**Comment by ignoramous** [comment]
Yeah, `serverless-dns` isn't setup in a special way for Fastly... This is a to-do.
